### PR TITLE
Update UG for Note search

### PR DESCRIPTION
**Pull request by aliciamichellew**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ SEP. It includes:
 - SEPlendid's myriad of features and commands
 - **Key parameters** to run SEPlendid smoothly
 
-If you are a new user, please head over to  [How to use the User Guide](#) to start planning for your study guide!
+If you are a new user, please head over to  [How to Use the User Guide](#) to start planning for your study guide!
```
Comment (lamchenghou) [inline docs/UserGuide.md:27]: Missing link?

**Review by lamchenghou — APPROVED** [review]
Looks good 👍

File: docs/UserGuide.md [hunk 2]
```diff
@@ -37,29 +37,29 @@ If you are a new user, please head over to  [How to use the User Guide](#) to st
...
-     - 4.2.2 [Add a partnercourse: `partnercourse add`](#4-2-2-add-a-partnercourse-partnercourse-add)
-     - 4.2.3 [Delete a partnercourse: `partnercourse delete`](#4-2-3-delete-a-partnercourse-partnercourse-delete)
-     - 4.2.4 [Search a partnercourse by attributes: `partnercourse search`](#4-2-4-search-a-partnercourse-by-attributes-partnercourse-search)
-     - 4.2.5 [Sort a partnercourse by attributes: `partnercourse sort`](#4-2-5-sort-a-partnercourse-by-attributes-partnercourse-sort)
+   - 4.2 [Commands for partner course](#4-2-commands-for-partnercourse)
+     - 4.2.1 [List all partner courses: `partnercourse list`](#4-2-1-list-all-partnercourses-partnercourse-list)
+     - 4.2.2 [Add a partner course: `partnercourse add`](#4-2-2-add-a-partnercourse-partnercourse-add)
+     - 4.2.3 [Delete a partner course: `partnercourse delete`](#4-2-3-delete-a-partnercourse-partnercourse-delete)
+     - 4.2.4 [Search a partner course by attributes: `partnercourse search`](#4-2-4-search-a-partnercourse-by-attributes-partnercourse-search)
+     - 4.2.5 [Sort a partner course by attributes: `partnercourse sort`](#4-2-5-sort-a-partnercourse-by-attributes-partnercourse-sort)
```
Comment (alyssaongyx) [inline docs/UserGuide.md:56]: good standardisation of words used in the ug

**Review by alyssaongyx — APPROVED** [review]
LGTM

**Pull request merged**